### PR TITLE
add --no-javascript option

### PR DESCRIPTION
**Pull request by pgrm**
Add option to disable javascript as discussed in https://github.com/arachnys/athenapdf/issues/85

### CODE REVIEW

**Comment by MrSaints** [comment]
Apologies for the delay, I'll look into this by the end of the week.

File: cli/src/athenapdf.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ athena
     .option("--no-portrait", "render in landscape")
     .option("--no-background", "omit CSS backgrounds")
     .option("--no-cache", "disables caching")
+    .option("--no-javasript", "disable JavaScript support")
```
Comment (MrSaints) [inline cli/src/athenapdf.js]: There's a typo here.

**Comment by MrSaints** [comment]
Hey, do you have time to look into the requested change? Otherwise, I'll close this PR in the next couple of days.

**Comment by pgrm** [comment]
hey sry - it's on my to do but at the very bottom at the moment. I'll close it for now and reopen once I'll have time to test it properly.

**Comment by MrSaints** [comment]
No worries, thanks a lot @pgrm 👍

**Pull request closed without merging**